src/components/UnsupportedBrowserNotice.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import {Dialog, DialogContent} from "@material-ui/core";
+import {withStyles} from "@material-ui/core";
+import {withCookies} from "react-cookie";
+import isSupported from "@mapbox/mapbox-gl-supported";
+
+const SEEN_UNSUPPORTED_BROWSER_BEFORE = "seenUnsupportedBrowserBefore";
+
+const styles = {
+  bulleted: {
+    listStyleType: 'disc'
+  }
+};
+
+const SplashPage = (props) => {
```

Comment:
Can we rename this to match the class name?

Reply:
yep for sure